client/src/components/login-signup/login.component.js

```diff
@@ -0,0 +1,71 @@
+import React, { Component } from "react";
```

Comment:
We should follow the standard naming convention for React component files: ComponentName.js. i.e. `Login.js`